apps/site/pages/en/learn/typescript/run-natively.md

```diff
@@ -22,10 +22,29 @@ node --experimental-strip-types example.ts
 
 And that's it! You can now run TypeScript code directly in Node.js without the need to transpile it first, and use TypeScript to catch type-related errors.
 
-In V22.7.0 this experimental support was extended to transform TypeScript-only syntax, like `enum`s and `namespace`, with the addition of the `--experimental-transform-types` flag.
+In V22.7.0 this experimental support was extended to transform TypeScript-only syntax, like `enum`s and `namespace`, with the addition of the `--experimental-transform-types` flag. Enabling `--experimental-transform-types` automatically implies that `--experimental-strip-types` is enabled, so there's no need to use both flags in the same command:
 
 ```bash
-node --experimental-strip-types --experimental-transform-types another-example.ts
+node --experimental-transform-types another-example.ts
+```
+
+From version V23 onwards, the `--experimental-strip-types` flag is enabled by default, enabling you to run any supported syntax, so running files like the one below with `node file.ts` is supported:
```

Comment:
Nit: "version V23" seems redundant; the V is already "version".

Reply:
applied the changes from @robpalme can you take a look again?

Reply:
I don't think he commented in this thread about "version V23", but the other changes seemed good.